.sandbox/provision/tf-lifecycle/main.tf

```diff
@@ -0,0 +1,96 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = ">=4"
+    }
+  }
+}
+
+data "external" "env" {
+  program = ["${path.module}/env.sh"]
+}
+
+provider "aws" {
+  default_tags {
+    tags = {
+      Sandbox   = data.external.env.result.sandbox_name
+      SandboxID = data.external.env.result.sandbox_id
+    }
+  }
+}
+
+data "aws_region" "current" {}
+
+data "aws_caller_identity" "current" {}
+
+resource "aws_ebs_volume" "data_volume" {
+  size              = 10
+  type              = "gp3"
```

Comment:
Could you make `size` and `type` from variables so it's configurable? This is a reference solution that it should be easy for people to directly use without modifying the code.